quad_pyblish_module/plugins/tvpaint/publish/extract_psd.py

```diff
@@ -64,7 +64,7 @@ def process(self, instance):
                 george_script_lines.append(
                     "tv_clipsavestructure \"{}\" \"PSD\" \"image\" {}".format(
                         dst_filepath,
```

Comment:
Maybe also using `dst_filepath.resolve()` in case of non absolute parts or symlinks.